helm-charts/basehub/values.yaml

```diff
@@ -580,27 +580,35 @@ jupyterhub:
         memory: 2Gi
     extraConfig:
       01-custom-theme: |
+        # adds a JupyterHub template path and updates template variables
+
         from z2jh import get_config
         c.JupyterHub.template_paths.insert(0,'/usr/local/share/jupyterhub/custom_templates')
         c.JupyterHub.template_vars.update({
             'custom': get_config('custom.homepage.templateVars')
         })
       02-custom-admin: |
+        # adjusts the spawner to add config specific to admin users
+
         from z2jh import get_config
         from kubespawner import KubeSpawner
         from jupyterhub_configurator.mixins import ConfiguratorSpawnerMixin
 
         class CustomSpawner(ConfiguratorSpawnerMixin, KubeSpawner):
             def start(self, *args, **kwargs):
                 custom_admin = get_config('custom.singleuserAdmin', {})
-                if custom_admin and self.user.admin:
-                    extra_init_containers = custom_admin.get('initContainers', [])
```

Comment:
Yes sorry for not documenting this, its part of [9672e0a](https://github.com/2i2c-org/infrastructure/pull/3039/commits/9672e0a08ee9211a7fe1cdb7bb572fdfa0135990) with a comment about it. The schema didn't allow this config to be passed via singleuserAdmin, so then nobody could have done so either, so I simply removed it thinking its not a likeley-to-be-used feature instead of fixing it.